### ISSUE TITLE
DuplicateRow() causes the parentheses to be lost in the formula of the row being duplicated

### ISSUE THREAD

**Issue by iciness** [qax-os/excelize#1914]
**Description**

DuplicateRow() causes the parentheses to be lost in the formula of the row being duplicated

**Steps to reproduce the issue:**
```
i := 2

for j := 0; j <= 10; j++ {	
	f.SetCellValue("Sheet2", "A"+strconv.Itoa(i), "1000")
	f.SetCellFormula("Sheet2", "B"+strconv.Itoa(i), "=(A"+strconv.Itoa(i)+"-800)*20%")
        f.DuplicateRow("Sheet2", i)
	i++
}
```

**Describe the results you received:**
After excel opens the file, the formula of A2 becomes = A2-800 * 20%
[Book1.xlsx](https://github.com/user-attachments/files/15532013/Book1.xlsx)

**Describe the results you expected:**
The brackets in the formula are not lost

**Output of `go version`:**

```
go version go1.22.3 windows/amd64
```

**Excelize version or commit ID:**

```
v2.8.1
```

**Environment details (OS, Microsoft Excel™ version, physical, etc.):**

windows 11 pro 23H2 22631.3672
execl  2405 Build 16.0.17628.20006 64bit

**Comment by iciness** [comment]
Add the following

**Comment by iciness** [comment]
Add the following
It seems to be caused by RemoveRow()
I has remove the duplicate of last line

**Comment by xuri** [comment]
Thanks for your feedback. This problem was duplicated with issue #1861, and already has been fixed in commit [`703b737`](https://github.com/xuri/excelize/commit/703b73779c06265bc8f23e3d9cbd628d6635fead). Please upgrade to the master branch code by `got get -u github.com/xuri/excelize/v2@master`, and this patch will be released in the next version. I closed this issue, if you have any questions, please let me know and you can reopen this anytime.
